.gitignore

```diff
@@ -1,3 +1,4 @@
 megaavr/tools/python3/*
 *.pyc
 *.pyc
+.DS_Store
```

Comment:
Do you have any idea why *.pyc is listed twice? 

And what is .DS_store?

Reply:
.DS_store is a garbage file that springs up on any repository touched by a Mac in every directory opened in the Mac version of explorer. It stores random metadata about icon positions and view modes. If not in git ignore you will get randomly committed .DS_Store files. No clue why pyc is there twice, remove one.